cli/cli.go

```diff
@@ -252,11 +257,6 @@ func getCliContext() *cli.App {
 		privateKeyFlag,
 		gcpKMSKeyFlag,
 		vaultTransitKeyFlag,
-		cli.StringSliceFlag{
-			Name: "script, s",
-			Usage: "Full path to the state script(s). You can specify multiple " +
-				"scripts providing this parameter multiple times.",
-		},
```

Comment:
It should also stay here, shouldn't it?

It's tempting to use the `artifactAddScripts` shared variable, but the help text is not appropriate for this location, since it is for new artifacts. So maybe it's best to avoid that and restore the old code.